tasks/task-arguments.js

```diff
@@ -1,4 +1,29 @@
-const argv = require('yargs').argv
-const destination = argv.destination ? argv.destination : 'public'
+const { argv } = require('yargs')
 
-exports.destination = destination
+// Defaults for known tasks
+const destinations = [
+  {
+    task: 'build:package',
```

Comment:
Part of me wants to do this by parsing the task name, but this is way less fussy and prone to errors!